src/app/zones/views/ZoneListView.vue

```diff
@@ -290,5 +281,7 @@ function toggleDeleteModal(row?: Record<string, any>) {
 
 function handleDelete() {
   toggleDeleteModal()
+
+  loadData(0)
```

Comment:
This is the bug fix, yes. We could tie logic like this more tightly into KTable and restructure our loadData functions to be full KTable fetchers und use the table fetcher key as the main "needs refetching" signal instead of calling loadData again, but that would be quite a bit of work and needed across multiple views to stay consistent. This is as simple as it gets for the time being.

Reply:
Ok then I think that this PR should only have this (and probably an e2e test to prove that we call the listing endpoint after a deletion, but I'll leave that decision to you).

The removal of all the watcher things can then be done across the entire application all in one go in a separate PR, I think? Wdyt?

Reply:
I dropped the extra commit, but please don’t ignore my requests of doing so when I ask it.

I consider adding a unit test for the extra request, but seeing that all our test environments use mocked data, getting something really reliable out of it is questionable.

Reply:
> I dropped the extra commit, but please don’t ignore my requests of doing so when I ask it.

I don't follow, did I miss something?

> I consider adding a unit test for the extra request, but seeing that all our test environments use mocked data, getting something really reliable out of it is questionable.

We can prove that the request is being made, which as far as I understand it was the problem? It doesn't matter what it responds with. There is a e2e test step just for this purpose. 

Anyway whether you want to add a test here or not is completely up to you. If you don't want to let me know and I can approve. As far as I understand it this line is all that required for this fix?